mkdocs.yml

```
@@ -122,8 +122,9 @@ nav:
     - Logger:                   reference/logger.md
     - Enviorment Variables:     reference/enviorment.md
```

Is not related maybe,, but here we can address the typo, also renaming the `.md` file.
```suggestion
    - Environment Variables:     reference/environment.md
```

---

I will take care of this in the environment variables #20 issue